sdk/template/aztemplate/ci.yml

```
@@ -0,0 +1,19 @@
+# NOTE: Please refer to https://aka.ms/azsdk/engsys/ci-yaml before editing this file.
+trigger:
+  paths:
+    include:
+    - sdk/template/aztemplate/
+    - eng/
+
+pr:
+  paths:
+    include:
+    - sdk/template/aztemplate/
+    - eng/
+
+
+stages:
+- template: ../../../eng/pipelines/templates/jobs/archetype-sdk-client.yml
+  parameters:
+    ServiceDirectory: 'aztemplate'
```

```suggestion
    ServiceDirectory: 'template/aztemplate'
```
For go ServiceDirectory needs to be the relative path under the sdk folder.